### DISCUSSION TITLE
dynamic routing trailing slashes is not routing correctly

### DISCUSSION THREAD

**Discussion by DeekshaPandit** [vercel/next.js#11402] [Help]
my directory is:

/pages/ 
   - [page1].js
   - anotherPage.js

The problem is when I do http://localhost:3000/anotherPage/  (look at trailing slash). 
It calls  [page1].js

Why trailing slash is calling another page?

**Comment by lewatt23** [comment]
@DeekshaPandit  please how did you solve it, i have the same problem..
